### PR TITLE
cxx cc extensions

### PR DESCRIPTION
**Pull request by sd0623**
- update to accept cxx and cc externsions
- but convert them to cpp (for markdown syntax and api call with language)

### CODE REVIEW

File: src/components/SubmitPage/SubmitCodeCard.tsx [hunk 1]
```diff
@@ -41,6 +41,8 @@ export default function SubmitCodeCard({
       'text/python': ['.py'],
       'text/c': ['.c'],
       'text/cpp': ['.cpp'],
```
Comment (AsirAAlam) [inline src/components/SubmitPage/SubmitCodeCard.tsx:43]: i wonder if it would work by passing them all in this array like `['.cpp', '.cxx', '.cc']`

**Review by AsirAAlam — APPROVED** [review]
Looks good! Just make sure to test it with all extensions. For example, to see if the syntax highlighting works.

**Comment by sd0623** [comment]
> Looks good! Just make sure to test it with all extensions. For example, to see if the syntax highlighting works.

Yeah, since I converting those extensions into cpp language, markdown syntax is working.

File: src/components/SubmitPage/SubmitCodeCard.tsx (hunk 1 above)
Comment (sd0623) [reply src/components/SubmitPage/SubmitCodeCard.tsx]: You are right. I was debugging and forgot to undo changes. But resolved now.

**Pull request merged**